Testsuite/test/run_testsuite_with_cmake

```diff
@@ -154,7 +154,11 @@ test_directory()
       START=`date +%s`
       run_local_cgal_test &
 
-      if wait_for_process "$!" "1200" "5"
+      TIMEOUT=1200
+      if [ "$1" = "Polyhedron_Demo" ]; then
+        TIMEOUT=5400
+      fi
+      if wait_for_process "$!" "$TIMEOUT" "5"
```

Comment:
If you have a look at the comments documenting `wait_for_process`, [above in the file][l], you can see that with `TIMEOUT=1200` the function `wait_for_process` waits 1200 times for 5 seconds (and that means 100 minutes). With  
`TIMEOUT=5400`, that would mean 450 minutes (7 hours and a half)! That seems too much.

  [l]: https://github.com/maxGimeno/cgal/blob/b7225bbf078ef2da1c79c228bba3ac855203a5c3/Testsuite/test/run_testsuite_with_cmake#L46

100 minutes is already a lot! How long need Visual Studio to compile the whole demo?! I think we should try first with just the double: `TIMEOUT=2400`